fjord/base/tests/test_auth.py

```diff
@@ -20,17 +22,18 @@ def test_new_user(self):
         # FIXME - this can go away post django-browserid 0.9
         new_user.backend = 'django_browserid.auth.BrowserIDBackend'
```

Comment:
How puzzling. Nah--let's just leave it there because it's a curious thing someone should look into some day.